docs/content.zh/docs/connectors/flink-sources/mongodb-cdc.md

```diff
@@ -333,31 +344,27 @@ MongoDB 的更改事件记录在消息之前没有更新。因此，我们只能
       <td>它指示在数据库中进行更改的时间。 <br>如果记录是从表的快照而不是改变流中读取的，该值将始终为0。</td>
     </tr>
     <tr>
-      <td>scan.incremental.snapshot.unbounded-chunk-first.enabled</td>
```

Comment:
This option is documented in the wrong place, so I move it from metadata column to connector option.